pysolarmanv5/pysolarmanv5.py

```diff
@@ -21,6 +23,15 @@
 _WIN_PLATFORM = platform.system() == "Windows"
 
 
+CONTROL = types.SimpleNamespace()
+CONTROL.HANDSHAKE = 0x41
+CONTROL.DATA = 0x42
+CONTROL.INFO = 0x43
+CONTROL.REQUEST = 0x45
+CONTROL.HEARTBEAT = 0x47
+CONTROL.REPORT = 0x48
```

Comment:
Until such time as we know what the second byte is used for, I propose leaving this field as two bytes. This is how the field is defined in the Solarman Java implementations.